src/Discovery/ConfiguredClientsStrategy.php

```diff
@@ -8,24 +8,14 @@
 use Http\Client\HttpClient;
 use Http\Discovery\HttpClientDiscovery;
 use Http\Discovery\Strategy\DiscoveryStrategy;
-use Symfony\Component\EventDispatcher\Event as LegacyEvent;
-use Symfony\Component\EventDispatcher\EventSubscriberInterface;
-use Symfony\Component\HttpKernel\Kernel;
-use Symfony\Contracts\EventDispatcher\Event;
-
-if (Kernel::MAJOR_VERSION >= 5) {
-    \class_alias(Event::class, 'Http\HttplugBundle\Discovery\ConfiguredClientsStrategyEventClass');
-} else {
-    \class_alias(LegacyEvent::class, 'Http\HttplugBundle\Discovery\ConfiguredClientsStrategyEventClass');
-}
```

Comment:
this was unused, correct?

Reply:
Yes, I've removed the `ConfiguredClientsStrategyEventClass $e` argument from the `onEvent()` method. The event was not used, so I figured the easiest method is to remove this "bc hack".